### PR TITLE
merge dev back in dsl2

### PR DESCRIPTION
**Pull request by maxulysse**
hopefully should fix lint

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,24 +49,24 @@ By default, the pipeline currently performs the following:
 
 ## Quick Start
 
-1. Install [`Nextflow`](https://nf-co.re/usage/installation)
+1. Install [`Nextflow`](https://nf-co.re/usage/installation) (`>=20.04.0`)
```
Comment (FriederikeHanssen) [inline README.md:52]: To match the batch, shouldn't this be: 
```suggestion
1. Install [`Nextflow`](https://nf-co.re/usage/installation) (`>=21.04.0`)
```

File: nextflow.config [hunk 2]
```diff
@@ -49,9 +49,11 @@ params {
...
   ascat_purity = null // Use default value
-  cf_coeff = "0.05"  // default value for Control-FREEC
-  cf_ploidy = "2"     // you can use 2,3,4
-  cf_window = null      // by default we are not using this in Control-FREEC
+  cf_coeff = 0.05                    // default value for Control-FREEC
+  cf_contamination = null            // by default not specified in Control-FREEC
+  cf_contamination_adjustment = null // by default we are not using this in Control-FREEC
+  cf_ploidy = "2"                    // you can use 2,3,4
+  cf_window = null                   // by default we are not using this in Control-FREEC
   generate_gvcf = null // g.vcf are not produced by HaplotypeCaller by default
```
Comment (FriederikeHanssen) [inline nextflow.config:57]: Seems like these parameters are added here but not applied anywhere so far. Is this intentional?

**Review by FriederikeHanssen — CHANGES_REQUESTED** [review]

File: nextflow.config (hunk 2 above)
Comment (maxulysse) [reply nextflow.config:57]: Yes, totally, so that I don't forget about all of them.

**Review by FriederikeHanssen — APPROVED** [review]
LGTM!

**Pull request merged**